hoomd/Integrator.cc

```diff
@@ -381,9 +385,9 @@ void Integrator::computeNetForce(uint64_t timestep)
                 }
 
             for (unsigned int k = 0; k < 6; k++)
-                external_virial[k] += (*force_compute)->getExternalVirial(k);
+                external_virial[k] += force->getExternalVirial(k);
```

Comment:
I avoid single-line if/for statements without enclosing braces. It's too error-prone and it always makes me feel "tricked" by the indentation.
```suggestion
                {
                external_virial[k] += force->getExternalVirial(k);
                }
```

Reply:
Sure, we can do this now. clang-format will take care of this when we get to implementing that.